### PR TITLE
feat(route): 番组计划用户想看

### PR DESCRIPTION
**Pull request by honue**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/bangumi/tv/user/wish/830573
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
  - [ ] EN / 英文文档
  - [ ] CN / 中文文档
- [ ] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/bangumi/tv/user/wish.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            const jdate = $(item).find('.collectInfo').find('span').html();
+            return {
+                title: aTag.html(),
+                link: 'https://bgm.tv' + aTag.attr('href'),
+                pubDate: timezone(parseDate(jdate), 0),
+            };
+        });
+
+    ctx.state.data = {
+        title: 'Bangumi 想看',
```
[the remark this answers is not in the document]
Comment (honue) [reply lib/v2/bangumi/tv/user/wish.js:32]: The website page title is "我想看的动画". Could you provide a description or more information?

[the remark this answers is not in the document]
Comment (honue) [reply lib/v2/bangumi/tv/user/wish.js:32]: Are you suggesting show the username in the title?

[the remark this answers is not in the document]
Comment (TonyRL) [reply lib/v2/bangumi/tv/user/wish.js:32]: Doesn't `$('head title').text()` works?

[the remark this answers is not in the document]
Comment (honue) [reply lib/v2/bangumi/tv/user/wish.js]: Of course, it works very well. However, the title is just '我想看的动画,' which doesn't match your description. I did little change.

**Pull request merged**